### PR TITLE
Login front end

### PR DESCRIPTION
**Pull request by audreynaidu**
Completed the login popup feature for the home page on the front end, allowing users to log in directly from a popup overlay after clicking the login button on the home page and makes a post request with the user credentials. It will need to be integrated with the backend so that the user can be properly authenticated.

### CODE REVIEW

**Review by cbollsandiego — COMMENTED** [review]
Everything looks good to me. We passed all of the unit tests and I ran through the acceptance tests on my end and everything works correctly. I am happy to merge this right now but am going to wait for another reviewer on the team to check off before we merge into main, as this should be our final steps to the completed project.

**Review by bilguuneo — APPROVED** [review]
Looks good, the login page looks acceptable and passes all the tests.

**Pull request merged**
